systemtests/src/main/java/com/github/streamshub/systemtests/constants/TimeConstants.java

```diff
@@ -16,6 +16,7 @@ private static long secondsInMilis(int seconds) {
     // Poll
     public static final long GLOBAL_POLL_INTERVAL_SHORT = secondsInMilis(3);
     public static final long POLL_INTERVAL_FOR_RESOURCE_READINESS = secondsInMilis(5);
+    public static final long ROLLING_UPDATE_POLL_INTERVAL = Duration.ofSeconds(5).toMillis();
```

Comment:
That's for all Pods in some subset? Is it enough?

Reply:
that is a poll interval not timeout, timeout is calculated later in the code